OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs

```diff
@@ -75,10 +75,40 @@ public void Run(ModData modData, string[] args)
 					}
 				}
 
+				var forbiddenBaseTypes = new[]
+				{
+					typeof(ITraitInfo),
+					typeof(TraitInfo<>),
+				};
+
+				var upgradableIndicator = "OpenRA.Mods.Common.Traits.UpgradableTraitInfo";
+				var upgradable = HasSuperType(t, upgradableIndicator);
+				doc.AppendFormat("Upgradable: {0}", upgradable ? "Yes" : "No");
+
+				var super = SuperType(t, forbiddenBaseTypes, new[] { upgradableIndicator });
+				if (super != null)
+				{
+					var superName = super.ToString();
+
+					// TODO: No idea why some types have this.
+					var bogus = superName.Contains("`");
+					if (!bogus)
+					{
+						doc.AppendLine();
+						var idx = superName.LastIndexOf(".");
+						superName = superName.Substring(idx + 1);
+
+						if (superName.EndsWith("Info"))
+							superName = superName.Substring(0, superName.Length - 4);
+
+						doc.Append("Subclass of [`{0}`](#{1}).".F(superName, superName.ToLower()));
```

Comment:
I suggest to add a comma: `, Subclass of...`

It is otherwise confusing to read: 
> ![image](https://cloud.githubusercontent.com/assets/756669/8043061/b09d7560-0e23-11e5-9636-70f3919f1fe5.png)